sqlglot/expressions.py

```diff
@@ -330,7 +330,13 @@ def __hash__(self):
 
 
 class Insert(Expression):
-    arg_types = {"this": True, "expression": True, "overwrite": False, "exists": False}
+    arg_types = {
+        "this": True,
+        "expression": True,
+        "overwrite": False,
+        "exists": False,
+        "partition": False,
```

Comment:
Make partition first class with 

{"keys": True, "values": False}

Reply:
or perhaps values true, i'm not sure on waht's cleaner

Reply:
created exp.Partition with just values. We do not need to track the keys and values separating and do zipping/unzipping.